core/src/main/scala/com/microsoft/azure/synapse/ml/causal/DiffInDiffEstimatorParams.scala

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.causal
+
+import org.apache.spark.ml.param.Params
+
+trait DiffInDiffEstimatorParams extends Params
```

Comment:
Can probabbly just put this in DiffInDiff file

Reply:
done.